### DISCUSSION TITLE
Samples for Wolverine Kafka

### DISCUSSION THREAD

**Discussion by ferdikosasih172147** [JasperFx/wolverine#1003] [Q&A] [answered]
Hi Team, 
I'm new using Wolverine Fx, and I would like to get a sample using Wolverine Kafka on :

1. How to publish messages with Kafka partition key
2. How to consume messages that are not produced by Wolverine Fx 

Thanks

**Comment by jeremydmiller** [comment, marked as answer]
I honestly didn't realize that we'd turned on Discussions for Wolverine.

Wolverine doesn't have any direct support for that *at this moment*. As I said on Discord as well, it's not a big change to add that. We've done similar things for AWS SQS and Azure Service Bus and it's not a lot of work. we could help you with the pull request if you were interested in doing that.

**Comment by ferdikosasih172147** [comment]
https://github.com/JasperFx/wolverine/pull/1008
